crates/holochain/src/conductor/config.rs

```diff
@@ -195,11 +200,45 @@ pub mod tests {
                     init_params: "some_params".into()
                 }),
                 passphrase_service: Some(PassphraseServiceConfig::Cmd),
+                keystore_path: None,
                 admin_interfaces: Some(vec![AdminInterfaceConfig {
                     driver: InterfaceDriver::Websocket { port: 1234 }
                 }]),
                 use_dangerous_test_keystore: true,
             }
         );
     }
+
+    #[test]
+    fn test_config_keystore() {
+        let toml = r#"
+    environment_path = "/path/to/env"
+    use_dangerous_test_keystore = true
+
+    keystore_path = "/path/to/keystore"
+
+    [passphrase_service]
+    type = "fromconfig"
+    passphrase = "foobar"
```

Comment:
Also might be that I'm missing context but this is just for testing right? We're not actually storing passwords in text files?

Reply:
We're totally storing passwords in text files : )

This is for the holoports - they need to be able to log into lair automatically when they reboot... so for now we're putting that unlock passphrase in the config file.

Someday it'd be nice to have a whole ecosystem where we could ping someone's connected cell phone when the holoport reboots and they could remotely enter the passphrase... but we're not there yet.